src/main/java/com/example/entity/Notes.java

```diff
@@ -0,0 +1,33 @@
+package com.example.entity;
+
+import javax.persistence.*;
```

Comment:
remove *. use only necessary imports

Reply:
@Vlada0612 If you think it's more productive, then okay.